src/control/common/proto/mgmt/pool.pb.go

```diff
@@ -6,8 +6,8 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1-devel
-// 	protoc        v3.5.0
+// 	protoc-gen-go v1.26.0
+// 	protoc        v3.19.4
```

Comment:
@wangshilong: Which distro are you using, and where did you get your protobuf tools? In Rocky 8.5, the standard version of protoc is 3.5.0. I think we need to all get on the same page with regard to protobuf tool versions so that we are not constantly hitting conflicts due to minor changes.

Reply:
I used centos7.9 for my local environment, shall we always use Rocky 8.5 for development?

Reply:
It's a good idea. :) CentOS 7.x is not supported upstream anymore and we are planning to drop support in DAOS as well. A big advantage to using Rocky 8.x for development is that the distro provides most of the tools we use, so you don't need to go find them from other places.